src/aadeshJun20/SwapNumbers.java

```diff
@@ -0,0 +1,21 @@
+package aadeshJun20;
+
+public class SwapNumbers {
+
+	void displaySwapping(int num1, int num2) {
+		System.out.println("Input number 1= " + num1 + ", number 2 = " + num2);
+		num1 = num1 + num2;
+		num2 = num1 - num2;
+		num1 = num1 - num2;
+		
+		System.out.println("output: num1= " + num1 + ", num2 = " + num2);
+	}
+
+	
+	public static void main(String[] args) {
+		SwapNumbers swapTwoNumbers = new SwapNumbers();
```

Comment:
Instead of line 16, 17 and 18 you can write it in one line:
new SwapNumbers().displaySwapping(200, 88);

Logic looks good to me.

Reply:
Changes done as per suggestion